reflex/components/component.py

```diff
@@ -435,30 +443,44 @@ def render(self) -> Dict:
             ),
             autofocus=self.autofocus,
         )
-        self._validate_component_children(
```

Comment:
should we keep the check here? Only reason is because if they somehow added children dynamically after creating their component, then we wouldn't catch it. Though I don't think I've seen people really do that...